core/commands/configure_network_command.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require_relative '../models/network_config_file'
+require_relative '../services/machine_configurator'
+require 'net/ssh'
+
+# This class loads ssh keys to configuration or selected nodes.
+class ConfigureNetworkCommand < BaseCommand
+  # Create the command instance.
+  # @param args [String] string of arguments splits / for the current command.
+  # @param keyFile [String] path for ssh key on local machine
+  # @param labels [String] label of nodes for the current command
+  # @param out [Out] the object that should be used to log information.
+  def initialize(args, env, out)
```

Comment:
You must not override the initialize method of the BaseCommand. The execution must begin within the execute method.